src/components/Block/Block.styles.ts

```diff
@@ -0,0 +1,64 @@
+/*! Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license. */
+import { BlockProps } from './Block';
+import { ellipsisStyle } from '../../util/styles';
+import { GutterSize } from '../FixedGrid/enums';
+import { textColors, fontSizes, lineHeights } from '../Text/Text.styles';
```

Comment:
these 3 mappings will be used throughout other components, should we move them to `util/styles/`?

Reply:
Yeah, I was a bit torn, I'll move them out and see how it looks.